src/engine/modules.js

```diff
@@ -179,7 +179,9 @@ function loadImports(self, imports) {
   }
   for (let i = 0; i < imports.length; ++i) {
     const [, moduleName, moduleVersion, moduleAlias] = imports[i];
-    if (typeof moduleVersion === "undefined") continue;
+    if (typeof moduleVersion === "undefined" || moduleVersion === null) {
```

Comment:
Perhaps just `!moduleVersion` (that would probably break module versions `0.0`, are those premitted in Qt?).